docs/SDK-GOVERNANCE.md

```diff
@@ -10,6 +10,23 @@ The community is organized as follows:
   [cloudevents/sdk-go project](https://github.com/cloudevents/sdk-go)
 - The union of all the `sdk-*-maintainers` assembles the _sdk maintainers_ group
 
+## SDK Repository Requirements
+
+Each SDK must include the following:
+- a pointer to it's github repo from the CloudEvents [README.md](README.md)
+  file
+- a README.md file with at least the following information:
+  - a description of the SDK
+  - a "Community" section with information about how to participate and
```

Comment:
Not sure that "how to participate" is required here when CONTRIBUTING.md is also required... although it would be good to link to that.